apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListFeatureExtractorSamples.kt

```diff
@@ -64,4 +64,49 @@ class ListFeatureExtractorSamples {
             }
         }
     }
+
+    @Test
+    fun lastFeature() {
+        val list = listOf(1, 2, 3)
+
+        expect(list).last.toEqual(3) // Subject is 3
+
+        expect(list).last // Subject is 3
+            .toBeGreaterThan(2) // Subject is still 3
+            .toBeLessThan(4) // Subject is still 3
+
+        fails {
+            expect(list).last.toBeGreaterThan(3).toBeLessThan (4) // Subject is 3, fails on first expectation, second is skipped
+        }
+
+        fails {
+            expect(listOf<Int>()).last.toEqual(3) // Fails, because list is empty
+        }
+    }
+
+    @Test
+    fun last() {
+        val list = listOf(1, 2, 3)
+
+        expect(list)
+            .last {
+                toEqual(3) // Subject is 3
+            }
+            .last { // Subject is 3
+                toBeGreaterThan(2) // Subject is still 3
+                toBeLessThan(4) // Subject is still 3
+            }
+
+        fails {
+            expect(list).last {
+                toBeGreaterThan(3).toBeLessThan (4) // Subject is 3, fails on first expectation, second is skipped
+            }
+        }
```

Comment:
second isn't skipped (no fail-fast behaviour within an expectation group block) and I suggest you copy from `get` above
```suggestion
        fails {
            // all expectations are evaluated inside an expectation-group block; for more details:
            // https://github.com/robstoll/atrium#define-single-expectations-or-an-expectation-group

            expect(list).last { // subject within this expectation-group is of type Int (actually 3)
                toBeGreaterThan(3)   // fails
                toBeLessThan (4)     // still evaluated, even though  `toBeGreaterThan` already fails,
                //                      use `.last.` if you want a fail fast behaviour
            } // subject here is back type List<Int>
        }
```